back-office/dashboard.md

```diff
@@ -0,0 +1,18 @@
+# **SPECIFICATIONS - DASHBOARD PAGE**
+
+
+_As a user, I want to know when my PHP version is no longer supported._
```

Comment:
It works also when PHP version is not **yet** supported. For example users trying to use php7.4 with PS 1.7.7.0 observe issues with menu.

We need users to use the right php version from PrestaShop (we provide a range): not lower, not higher.

Reply:
```suggestion
_As a user, I want to know when my PHP version is not suitable - no longer supported or not yet supported._
```

Reply:
How the message be displayed for the 1.7.7.0 version since the feature is integrated to the 1.7.8.0 ? @matks